tests/integration/components/resize-sensor-test.js

```diff
@@ -0,0 +1,60 @@
+import { htmlSafe } from "@ember/string";
+import { later } from "@ember/runloop"
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import wait from 'ember-test-helpers/wait';
+
+moduleForComponent('resize-sensor', 'Integration | Component | resize sensor', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  assert.expect(2);
+
+  this.render(hbs`{{resize-sensor}}`);
+
+  assert.equal(this.$().text().trim(), '');
+
+  // Template block usage:
+  this.render(hbs`
+    {{#resize-sensor}}
+      template block text
+    {{/resize-sensor}}
+  `);
+
+  assert.equal(this.$().text().trim(), 'template block text');
+});
+
+test('when the target changes sizes it will trigger the action', function(assert){
+  
+    assert.expect(1);
+  
+    this.set('style', htmlSafe('width: 500px; height: 500px;'));
+  
+    let updated;
+    this.on('resized', function(element){
+      updated = {
+        width: element.offsetWidth,
+        height: element.offsetHeight
+      }
+    });
+  
+    this.render(hbs`
+    <div style={{style}}>
+      {{#resize-sensor onResize='resized'}}
+      {{/resize-sensor}}
+    </div>
+    `);
+  
+    this.set('style', htmlSafe('width: 250px; height: 250px;'));
+  
+    later(() => {
+      assert.deepEqual(updated, { width: 250, height: 250 }, 'received updated size');
```

Comment:
This assert can just be in the `this.on('resized'` action, that we we don't need to use `run.later` and rely on timing